_includes/manuals/1.19/4.3.2_smartproxy_settings.md

```diff
@@ -0,0 +1,124 @@
+The main configuration for the core Smart Proxy is held in the */etc/foreman-proxy/settings.yml* or *config/settings.yml* file.  This includes configuration of ports to listen on, SSL and security settings and logging options.
+
+Each of the modules used in the Smart Proxy have their configuration in the */etc/foreman-proxy/settings.d/* or *config/settings.d* directory.  Modules are enabled or disabled inside their respective configuration files with the `:enabled` directive, which determines whether the module is available on HTTP, HTTPS, both or is disabled (see below for more details).
+
+#### YAML start
+
+The first non-comment line of all configuration files must be three dashes.
+
+<pre>---</pre>
+
+#### Daemon configuration (settings.yml)
+
+If daemon is present and true then the Smart Proxy will attempt to disconnect itself from the controlling terminal and daemonize itself on startup, writing its pid (process ID) into the specified file.
+
+<pre>
+:daemon: true
+:daemon_pid: /var/run/foreman-proxy/foreman-proxy.pid
+</pre>
+
+#### Logging (settings.yml)
+
+The proxy's output is captured to the the **log_file** and may be filtered via the usual Unix syslog levels:
+
+* *WARN*
+* *DEBUG*
+* *ERROR*
+* *FATAL*
+* *INFO*
+* *UNKNOWN*
+
+See Ruby's [Logger class](http://www.ruby-doc.org/stdlib/libdoc/logger/rdoc/classes/Logger.html) for details.
+
+<pre>
+:log_file: /var/log/foreman-proxy/proxy.log
+:log_level: DEBUG
+</pre>
+
+The log_file setting may be set to "STDOUT" which causes log messages to be logged to standard output, for capture by the running process (e.g. systemd with journal).  When log_file is set to "SYSLOG", all messages will be sent to syslog.
+
+A limited number of recent log messages are kept in memory using a ring buffer, which can be exposed in the API and to Foreman by enabling the [Logs feature](/manuals/{{page.version}}/index.html#4.3.13Logs).
+
+The number of all log messages is controlled by the `log_buffer` setting, and a second buffer of error messages is controlled by the `log_buffer_errors` setting.  The total of the two will directly affect the maximum amount of memory used, which is approximately 500kB in the default configuration of 3,000 recent messages.
+
+<pre>
+:log_buffer: 2000
+:log_buffer_errors: 1000
+</pre>
+
+#### Listening configuration (settings.yaml)
+
+By default the Smart Proxy listens on all interfaces, which can be changed to limit access to a network:
+
+<pre>
+# host to bind ports to (possible values: *, localhost, 0.0.0.0)
```

Comment:
Here we need a note that on EL7 we now default to `::` but if you have disabled IPv6 at the kernel level then you must change it to `*`.